Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Tasks](https://hosted.weblate.org/projects/kimai/tasks/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Actions (Dropdowns and Toolbars)](https://hosted.weblate.org/projects/kimai/actions/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kimai/messages/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/kimai/kimai?pullRequest=4862) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/kimai/kimai?pullRequest=4862) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/kimai/kimai?pullRequest=4862) it.</sub>

weblate committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (37 of 37 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (78 of 78 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (27 of 27 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (436 of 436 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (78 of 78 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (27 of 27 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (436 of 436 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (78 of 78 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (37 of 37 strings)

Translated using Weblate (Indonesian)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Ukrainian)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Danish)

Currently translated at 41.6% (5 of 12 strings)

Translated using Weblate (Croatian)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Greek)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Persian)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Polish)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Czech)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Chinese (Simplified))

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Arabic)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Slovak)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Korean)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Spanish)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Finnish)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Turkish)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Russian)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Portuguese)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Hungarian)

Currently translated at 75.0% (9 of 12 strings)

Translated using Weblate (French)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Vietnamese)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Swedish)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Romanian)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Dutch)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (Italian)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Hebrew)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Esperanto)

Currently translated at 66.6% (8 of 12 strings)

Translated using Weblate (German)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (English)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (436 of 436 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (436 of 436 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Milo Ivir <mail@milotype.de>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>
Co-authored-by: SC <lalocas@protonmail.com>
Co-authored-by: Wei-Cheng Yeh (IID) <iid@ccns.ncku.edu.tw>
Co-authored-by: Wellington Terumi Uemura <wellingtonuemura@gmail.com>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/kimai/daterangepicker/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/de_CH/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/el/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/en/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/eo/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/es/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/fa/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/fi/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/hu/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/ko/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/nl/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/pl/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/pt/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/ro/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/ru/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/sk/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/sv/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/tr/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/uk/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/vi/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/export/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/reporting/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/tasks/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/zh_Hant/
Translation: Kimai - Time tracking/Date-Picker
Translation: Kimai - Time tracking/Emails
Translation: Kimai - Time tracking/Export
Translation: Kimai - Time tracking/Invoice-Templates
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Reporting
Translation: Kimai - Time tracking/System configurations
Translation: Kimai - Time tracking/Tasks
Translation: Kimai - Time tracking/Validations
